src/compiler/prioritization/priority-levels.md

```diff
@@ -0,0 +1,96 @@
+
+  - Mitigating conditions that may lower priority:
+    - If the code should never have compiled in the first place (but if the regression affects a large number of crates, this may indicate that we need a warning period)
+    - If the code in question is theoretical and considered unlikely to exist in the wild, or if it only exists in small, unmaintained packages that are not widely used
+  - If a regression has been in stable for a release or two (either because we are still awaiting a fix, or because the bug had laid dormant i.e. undetected), we typically lower the priority as well, because by that time, if the users have not raised a ruckus about the regression, that is a sign that it is inherently not a critical issue. Eg: [an issue that would have been P-Critical but ended up being P-High](https://rust-lang.zulipchat.com/#narrow/stream/227806-t-compiler.2Fwg-prioritization/topic/pre-meeting.20triage.202020-04-09.20.2354818)
+
+- Regressions where code still compiles but does something different than it used to do (dynamic semantics have changed)
+  - Mitigating conditions that may lower priority:
+    - If code uses feature that is explicitly not specified (e.g. `std::vec::Vec` docs state order in which it drops its elements is subject to change)
+- Feature-gated features accessible without a feature gate
+  - Mitigating conditions that may lower priority:
+    - If the pattern is VERY unlikely
+- Soundness holes where common code that should not compile actually does
+  - Mitigating conditions that may lower priority:
+    - Soundness holes that are difficult to trigger
+    - Soundness holes that have been around for a very long time may be critical, but typically require
+- Diagnostic regressions where the diagnostic is very common and the situation very confusing
+- ICEs for common scenarios or code patterns
+  - Mitigating conditions that may lower priority:
+    - If the code that triggers the ICE also triggers compilation errors, and those errors are emitted before the ICE
+    - If the code in question makes use of unstable features, particularly if the ICE requires a feature gate
+
+### P-High
+
+`P-High` issues are the ones the compiler team will focus on the most.
+
+They can be `P-Critical` issues that have a mitigating condition as defined above, or important issues that aren't deemed blockers.
+
+The effectiveness of the prioritization Working Group will be a direct consequence of our ability to draw the line between `P-Critical` and `P-High` issues.
+
+### P-Medium
+
+`P-Medium` refer to issues that aren't critical, and that will be resolved in the long run. Eg issues that will be fixed after a specific feature has landed.
+
+### P-Low
+
+`P-Low` bugs are bugs that don't cause soundness or safety issues, and that occur in a "failing path" already. Eg: Code that wouldn't compile and raises an error. It might also refer to features that would be nice to have, but aren't deemed a priority
```

Comment:
I'm not loving the definition of P-low we have here. I think we've documented the different levels in a different place elsewhere? maybe we can reuse some explanations from other documents. Don't remember to be honest what we have. cc @rust-lang/wg-prioritization

Reply:
I might have seen a definition like "issue that the compiler team doesn't plan to resolve, but are still worth fixing".